script.js

```diff
@@ -3,24 +3,141 @@ const hamburgerImg = document.querySelector('#hamburger-img');
 const navMenu = document.querySelector('#hamburger-menu');
 const menuOptions = navMenu.querySelectorAll('a');
 const body = document.querySelector('body');
-const form = document.querySelector('.contact-form');
-const emailInput = document.querySelector('#email');
-const errorMsg = document.querySelector('#error-message');
-
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-  if (emailInput.value !== emailInput.value.toLowerCase()) {
-    errorMsg.textContent = 'Please enter your email in all lowercase!';
-  } else {
-    errorMsg.textContent = '';
-    form.submit();
-  }
+const portfolioGrid = document.querySelector('#portfolio-grid');
+const popup = document.querySelector('#popup');
+const workBtn = document.querySelector('.work-btn');
+const saveInfo = document.querySelector('.save-info');
+
+const dataList = JSON.parse(localStorage.myFormList);
```

Comment:
- [ ] You should use a getItem and this is the reson it throws the error as indicated in the attached image.
![image](https://user-images.githubusercontent.com/86200356/202476068-99c9eeb9-d48f-4737-b281-bae471de25ba.png)

Reply:
Thank you for your feedback. Corrections has been made.